CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Changelog
+# 1.84.0
+* Enhancement - CannotPullContainerError/ASM error message enhancements [#4181](https://github.com/aws/amazon-ecs-agent/pull/4181)
+* Enhancement - update go to 1.22.3 [#4184](https://github.com/aws/amazon-ecs-agent/pull/4184)
+* Enhancement - cap cpu shares to the max allowed value on Linux [#4204](https://github.com/aws/amazon-ecs-agent/pull/4204)
+* Bugfix - S3 config changes to use S3 FIPS endpoints on FIPS hosts [#4202](https://github.com/aws/amazon-ecs-agent/pull/4202)
```

Comment:
nit: Should this be considered a Bugfix? I was under the impression this is a new enhancement that we are adding.

Reply:
Synced up offline, pull request author confirmed this is a bugfix.